package.json

```diff
@@ -78,21 +76,18 @@
     "lerna-parallelism": "^0.3.0",
     "lint-staged": "^10.5.0",
     "mendix-client": "^7.15.8",
-    "node-fetch": "^2.6.1",
-    "node-ip": "^0.1.2",
     "prettier": "^2.5.1",
     "pretty-quick": "^3.1.0",
-    "react": "~17.0.1",
```

Comment:
Why we removed this deps?

Reply:
It is already inside pluggable widgets tools, so its irrelevant in the top level.